Client.Core/Flux/Internal/FluxCsvParser.cs

```diff
@@ -267,7 +267,14 @@ private FluxRecord ParseRecord(int tableIndex, FluxTable table, CsvReader csv)
 
                 var strValue = csv[fluxColumn.Index + 1];
 
+                if (record.Values.ContainsKey(columnName))
+                {
+                    record.Values.Remove(columnName);
+                }
+
                 record.Values.Add(columnName, ToValue(strValue, fluxColumn));
```

Comment:
We can simplify the code to: `record.Values[columnName] = ToValue(strValue, fluxColumn);`